task_schedule.cfg

```diff
@@ -43,7 +43,7 @@ alert       aca@head.cfa.harvard.edu
       exec 1: parse_cmd_load_gen.pl --dbi 'sybase' --server 'sybase' --verbose
       exec 1: update_load_seg_db.py --dbi 'sybase' --server 'sybase' --verbose
       exec 6: $ENV{SKA_SHARE}/cmd_states/update_cmd_states.py --dbi 'sybase' --server 'sybase' --h5file $ENV{SKA_DATA}/cmd_states/cmd_states.h5
-      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --verbose
+      exec 1: parse_cmd_load_gen.pl --dbi 'sqlite' --server $ENV{SKA_DATA}/cmd_states/cmd_states.db3 --touch_file $ENV{SKA_DATA}/timelines/sqlite_parsed_files.touch  --verbose 
```

Comment:
Got it.  In that case can you rename the touch file to be `sum_files_sqlite3.touch` for consistency with the sybase one?  (I.e. so when I `ls` that directory they are next to each other and contextually connected).